projects/observability/src/pages/explorer/explorer.component.ts

```diff
@@ -325,22 +324,32 @@ export class ExplorerComponent {
     ];
   }
 
-  private tryDecodeExploreOrderBy(orderByString?: string): ExploreOrderBy | undefined {
+  private tryDecodeExploreOrderBy(
+    interval: IntervalValue,
+    selectedSeries: ExploreSeries,
+    orderByString?: string
+  ): ExploreOrderBy | undefined {
     const matches = orderByString?.match(/(\w+)\((\w+)\):(\w+)/);
 
     if (matches?.length !== 4) {
+      if (interval === 'NONE') {
```

Comment:
If the interval is not none, the whole thing is undefined in any case. We could put this as the first line of this function or better yet, outside it. Only try to decode order by if interval is not none e.g. `orderBy: interval === 'NONE' ? this.tryDecodeExploreOrderBy(series[0], param.get(ExplorerQueryParam.Order)) : undefined`